### PR TITLE
Quick Updates + Bugs Fixing

### PR DESCRIPTION
**Pull request by KitsuneModder**
Esses ''Quick Updates'' são pequenos updates que foram feitos ao jogo, enquanto no beta (1.3).

Os ''Bugs Fixing'' são correções de erros que apareceram no registro ao iniciar o RimWorld (1.3).

### "Erros"
`Couldn't inject ThreatReward_Manhunters_Joiner.LetterLabelChasing.slateRef (ThreatReward_Manhunters_Joiner.root.nodes.7.node.nodes.0.customLetterLabel.slateRef) into RimWorld.QuestScriptDef (Scripts_JoinerThreat.xml): Field or TKey nodes does not exist.
Couldn't inject ThreatReward_Manhunters_Joiner.LetterTextChasing.slateRef (ThreatReward_Manhunters_Joiner.root.nodes.7.node.nodes.0.customLetterText.slateRef) into RimWorld.QuestScriptDef (Scripts_JoinerThreat.xml): Field or TKey nodes does not exist.
Couldn't inject ThreatReward_MysteryThreat_Joiner.LetterLabelChasingRaid.slateRef (ThreatReward_MysteryThreat_Joiner.root.nodes.6.node.nodes.0.nodes.0.nodes.1.value.slateRef) into RimWorld.QuestScriptDef (Scripts_JoinerThreat.xml): Field or TKey nodes does not exist.
Couldn't inject ThreatReward_MysteryThreat_Joiner.LetterTextChasingRaid.slateRef (ThreatReward_MysteryThreat_Joiner.root.nodes.6.node.nodes.0.nodes.0.nodes.2.value.slateRef) into RimWorld.QuestScriptDef (Scripts_JoinerThreat.xml): Field or TKey nodes does not exist.
Couldn't inject ThreatReward_MysteryThreat_Joiner.LetterLabelChasingInfestation.slateRef (ThreatReward_MysteryThreat_Joiner.root.nodes.6.node.nodes.0.nodes.1.nodes.0.value.slateRef) into RimWorld.QuestScriptDef (Scripts_JoinerThreat.xml): Field or TKey nodes does not exist.
Couldn't inject ThreatReward_MysteryThreat_Joiner.LetterTextChasingInfestation.slateRef (ThreatReward_MysteryThreat_Joiner.root.nodes.6.node.nodes.0.nodes.1.nodes.1.value.slateRef) into RimWorld.QuestScriptDef (Scripts_JoinerThreat.xml): Field or TKey nodes does not exist.
Couldn't inject ThreatReward_MysteryThreat_Joiner.LetterLabelSiteAppeared.slateRef (ThreatReward_MysteryThreat_Joiner.root.nodes.6.node.nodes.0.nodes.2.nodes.1.value.slateRef) into RimWorld.QuestScriptDef (Scripts_JoinerThreat.xml): Field or TKey nodes does not exist.
Couldn't inject ThreatReward_MysteryThreat_Joiner.LetterTextSiteAppeared.slateRef (ThreatReward_MysteryThreat_Joiner.root.nodes.6.node.nodes.0.nodes.2.nodes.2.value.slateRef) into RimWorld.QuestScriptDef (Scripts_JoinerThreat.xml): Field or TKey nodes does not exist.
Couldn't inject ThreatReward_MysteryThreat_Joiner.LetterLabelChasingManhunter.slateRef (ThreatReward_MysteryThreat_Joiner.root.nodes.6.node.nodes.0.nodes.3.nodes.0.value.slateRef) into RimWorld.QuestScriptDef (Scripts_JoinerThreat.xml): Field or TKey nodes does not exist.
Couldn't inject ThreatReward_MysteryThreat_Joiner.LetterTextChasingManhunter.slateRef (ThreatReward_MysteryThreat_Joiner.root.nodes.6.node.nodes.0.nodes.3.nodes.1.value.slateRef) into RimWorld.QuestScriptDef (Scripts_JoinerThreat.xml): Field or TKey nodes does not exist.
Couldn't inject ThreatReward_RaidMultiFaction_Joiner.LetterLabelChasingRaid1.value.slateRef (ThreatReward_RaidMultiFaction_Joiner.root.nodes.10.node.nodes.0.parms.5.value.slateRef) into RimWorld.QuestScriptDef (Scripts_JoinerThreat.xml): Field or TKey nodes does not exist.
Couldn't inject ThreatReward_RaidMultiFaction_Joiner.LetterTextChasingRaid1.value.slateRef (ThreatReward_RaidMultiFaction_Joiner.root.nodes.10.node.nodes.0.parms.6.value.slateRef) into RimWorld.QuestScriptDef (Scripts_JoinerThreat.xml): Field or TKey nodes does not exist.
Couldn't inject ThreatReward_RaidMultiFaction_Joiner.LetterLabelChasingRaid2.value.slateRef (ThreatReward_RaidMultiFaction_Joiner.root.nodes.10.node.nodes.1.node.nodes.0.parms.4.value.slateRef) into RimWorld.QuestScriptDef (Scripts_JoinerThreat.xml): Field or TKey nodes does not exist.
Couldn't inject ThreatReward_RaidMultiFaction_Joiner.LetterTextChasingRaid2.value.slateRef (ThreatReward_RaidMultiFaction_Joiner.root.nodes.10.node.nodes.1.node.nodes.0.parms.5.value.slateRef) into RimWorld.QuestScriptDef (Scripts_JoinerThreat.xml): Field or TKey nodes does not exist.
Couldn't inject ThreatReward_SiteThreat_Joiner.LetterTextSiteAppeared.value.slateRef (ThreatReward_SiteThreat_Joiner.root.nodes.6.node.nodes.0.parms.3.value.slateRef) into RimWorld.QuestScriptDef (Scripts_JoinerThreat.xml): Field or TKey nodes does not exist.`

Esses estão com problema.

Verifiquei no servidor oficial de desenvolvimento do RimWorld.

### CODE REVIEW

File: Core/DefInjected/InstructionDef/Instructions.xml [hunk 1]
```diff
@@ -97,13 +97,13 @@
   <!-- EN: Welcome to RimWorld!\n\nThis page configures how your world should be generated.\n\nThe default settings here are good - go ahead and click GENERATE! -->
   <CreateWorldParams.text>Bem-vindo ao Rimworld!\n\nEsta página configura como seu mundo deve ser gerado.\n\nAs configurações padrão aqui são boas - vá em frente e clique em GERAR!</CreateWorldParams.text>
   <!-- EN: Click GENERATE to generate your world. -->
-  <defName.rejectInputMessage>Clique em GERAR para gerar seu mundo.</defName.rejectInputMessage>
+  <rejectInputMessage.text>Clique em GERAR para gerar seu mundo.</rejectInputMessage.te>
```
Comment (LoboMetalurgico) [inline Core/DefInjected/InstructionDef/Instructions.xml:100]: acredido que deveria ser </rejectInputMessage.text>

File: Royalty/Keyed/Misc_Gameplay.xml [hunk 2]
```diff
@@ -165,6 +165,17 @@
   <!-- EN: Bestowing ceremony total quality was {0}, determined by these factors -->
   <BestowingOutcomeQualitySpecific>A qualidade total da cerimônia de outorga era {0}, determinada por esses fatores</BestowingOutcomeQualitySpecific>
 
+  <!-- EN: No colonists that can link with anima tree right now. -->
+  <RitualTargetAnimaTreeNoPawnsToLink>Nenhum colonizo pode se conectar com a árvore anima agora.</RitualTargetAnimaTreeNoPawnsToLink>
```
Comment (LoboMetalurgico) [inline Royalty/Keyed/Misc_Gameplay.xml:169]: não seria melhor usar "colono" ao invés de "colonizo"?

**Review by LoboMetalurgico — CHANGES_REQUESTED** [review]
Olá, obrigado pelo seu pull request.

Além dos comentários indicando coisas para serem corrigidas, tenho uma pergunta:

Em alguns lugares, "mote" está escrito igual o inglês, mas em outros, está como "partícula". Porque?

File: Core/DefInjected/InstructionDef/Instructions.xml (hunk 1 above)
Comment (KitsuneModder) [reply Core/DefInjected/InstructionDef/Instructions.xml:100]: Já está corrigido:)

File: Royalty/Keyed/Misc_Gameplay.xml (hunk 2 above)
Comment (KitsuneModder) [reply Royalty/Keyed/Misc_Gameplay.xml:169]: Corrigido

**Comment by KitsuneModder** [comment]
Sobre o Mote/Partícula,

Eu também não sei.

Apenas deixei como estava...

**Comment by KitsuneModder** [comment]
Como lançou o Ideology

Ele o traduzurei :)

Mas em outro pull request

**Pull request merged**